cmd/dbmigrate/main.go

```diff
@@ -119,49 +72,32 @@ func main() {
 		os.Exit(1)
 	}
 
-	err = openDbConnection(conf.Database.Driver)
-	if err != nil {
-		fmt.Println(err)
+	if !conf.Database.Enabled {
+		fmt.Println("Database support is disabled")
 		os.Exit(1)
 	}
-
-	fmt.Printf("Connected to: %s\n", conf.Database.Host)
-
-	temp.Conn = dbConn
-
-	err = temp.RunMigration()
+	err = openDbConnection(conf.Database.Driver)
 	if err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
 
-	if dbConn.SQL != nil {
-		err = dbConn.SQL.Close()
-		if err != nil {
-			fmt.Println(err)
-		}
-	}
-}
+	drv := repository.GetSQLDialect()
 
-func newMigrationFile(filename string) error {
-	curTime := strconv.FormatInt(time.Now().Unix(), 10)
-	path := filepath.Join(migrationDir, curTime+"_"+filename+".sql")
-	err := common.CreateDir(migrationDir)
-	if err != nil {
-		return err
+	if drv == database.DBSQLite || drv == database.DBSQLite3 {
+		fmt.Printf("Database file: %s\n", conf.Database.Database)
+	} else {
+		fmt.Printf("Connected to: %s\n", conf.Database.Host)
 	}
-	fmt.Printf("Creating new empty migration: %v\n", path)
-	f, err := os.Create(path)
 
-	if err != nil {
-		return err
+	if command == "" {
+		_ = goose.Run("status", dbConn.SQL, drv, migrationDir, "")
+		fmt.Println()
+		flag.Usage()
+		return
 	}
 
-	_, err = f.Write(defaultMigration)
-
-	if err != nil {
-		return err
+	if err = goose.Run(command, dbConn.SQL, drv, migrationDir, args); err != nil {
```

Comment:
make sure when we go up/ down /reset we print the file we used for migrations so its more obvious on workflow. Like:
![image](https://user-images.githubusercontent.com/9391715/66012191-87f5a580-e509-11e9-924b-cf055607dc69.png)

Reply:
Will need to make this change to Goose as the output comes from there

Reply:
Change has been made to the [Goose](https://github.com/thrasher-corp/goose) package same output format as status